database/seeders/UserSeeder.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Database\Seeders;
+
+use Domain\Users\Models\User;
+use Illuminate\Database\Seeder;
+
+class UserSeeder extends Seeder
+{
+    public function run(): void
+    {
+        User::factory()->create([
+            'name' => 'Admin',
+            'email' => 'admin@ascend.agency',
```

Comment:
```suggestion
            'email' => 'developers@ascend.agency',
```

non-blocking: Traditionally for all products we've used this email address for access

Reply:
Nice, thank you